fgpyo/vcf/tests/test_builder.py

```diff
@@ -81,9 +81,11 @@ def _get_random_variant_inputs(
         key: (
             random_generator.randint(0, 100)
             if value_type == VcfFieldType.INTEGER
-            else random_generator.uniform(0, 1)
-            if value_type == VcfFieldType.FLOAT
-            else random_generator.choice(["Up", "Down"])
+            else (
+                random_generator.uniform(0, 1)
+                if value_type == VcfFieldType.FLOAT
+                else random_generator.choice(["Up", "Down"])
+            )
```

Comment:
Same section we talked about yesterday. I'm not sure if this is black saying "no really, I insist" or if this change just predated the revert on the previous PR. If it's easy and you don't mind reverting, I'd prefer it. Otherwise it's fine..